examples/platform/silabs/BaseApplication.cpp

```diff
@@ -188,6 +188,17 @@ void BaseApplicationDelegate::OnCommissioningWindowClosed()
         }
     }
 #endif // CHIP_CONFIG_ENABLE_ICD_SERVER && SLI_SI917
+    if (BaseApplication::GetProvisionStatus())
```

Comment:
Is `OnCommissioningWindowClosed` the best location for this screen change?
The API is called irrespective of if the commissioning has been succesful or not.

If we opt to keep it here, we should add the same code to transition to the QR code when the commissioning window is opened.

Reply:
`BaseApplication::GetProvisionStatus()` would be indicative of a successful provision, right? But I see a point in having the QR code screen to be tied up to commissioning window open. @jmartinez-silabs your thoughts ?